packages/dev/core/src/Meshes/Builders/groundBuilder.ts

```diff
@@ -397,8 +405,9 @@ export function CreateGroundFromHeightMap(
         colorFilter?: Color3;
         alphaFilter?: number;
         updatable?: boolean;
-        onReady?: (mesh: GroundMesh) => void;
+        onReady?: (mesh: GroundMesh, heightBuffer: Float32Array) => void;

```

Comment:
heighBuffer should be optional in the callback, otherwise existing onReady implementations will show an error. only pass it if the flag is true